### PR TITLE
Development

### PR DESCRIPTION
**Pull request by YemreAybey**

### CODE REVIEW

File: enumarable.rb [hunk 1]
```diff
@@ -0,0 +1,194 @@
+module Enumarable
```
Comment (inhaq) [inline enumarable.rb]: typo `Enumerable`

File: enumarable.rb [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+          when true then return false
+        end
+      end
+    else
+      return "You can't use my_none? method on #{self.class} class"
+    end
+    true
+  end
+  
+  def my_count
```
Comment (inhaq) [inline enumarable.rb]: `my_count` should work with and without block

**Review by inhaq — CHANGES_REQUESTED** [review]
Good job, just minor updates required.
Kindly follow comments and update the code accordingly. Once updated, resubmit for review.

File: enumarable.rb [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+    acc
+  end
+  
+    def multiply_els
+      self.my_inject {|acc, num| acc*num}
+    end
+  
+  end
+
+  include Enumerable
```
Comment (inhaq) [inline enumarable.rb]: `include Enumerable`not needed here, remove next 2 lines as well

**Review by inhaq — APPROVED** [review]
Awesome, it's 👍🏻
Before merging, fix a minor issue in the comment.
Good luck with the next project.

File: enumarable.rb (hunk 3 above)
Comment (YemreAybey) [reply enumarable.rb]: Thanks Inhaq for your efforts

**Pull request merged**